pom.xml

```diff
@@ -6,7 +6,7 @@
   <!-- Arifact name and version information -->
   <groupId>net.snowflake</groupId>
   <artifactId>snowflake-ingest-sdk</artifactId>
-  <version>2.0.1</version>
+  <version>2.0.2-SNAPSHOT</version>
```

Comment:
requestbuilder.java also has version which IMO we should update. It is sent as user agent in our rest endpoints. 
It will help us distinguish on server side.

Reply:
+1, please see the previous commit as an example